lambda_function.py

```diff
@@ -2,6 +2,7 @@
 import json
```

Comment:
can we remove this import now that we are not using it and using the dynamodb_json library instead

Reply:
done